app.js

```diff
@@ -2,18 +2,25 @@ const htmlStandards = require('reshape-standard')
 const cssStandards = require('spike-css-standards')
 const jsStandards = require('spike-js-standards')
 const pageId = require('spike-page-id')
+const sugarml = require('sugarml')
+const sugarss = require('sugarss')
 const products = require('./assets/data/products/index.js')
+const isProduction = process.env.SPIKE_ENV === 'production'
```

Comment:
When you compile with `spike -e production` (or whatever other environment, it will set a `SPIKE_ENV` environment variable automatically with the name of the environment. In this case, we can get this option and use it to easily configure minification etc.